lib/experimental/fonts/font-library/class-wp-rest-font-collections-controller.php

```diff
@@ -95,17 +146,126 @@ public function get_item( $request ) {
 			return $collection;
 		}
 
-		$config   = $collection->get_config();
-		$contents = $collection->get_content();
+		$item = $this->prepare_item_for_response( $collection, $request );
 
-		// If there was an error getting the collection data, return the error.
-		if ( is_wp_error( $contents ) ) {
-			$contents->add_data( array( 'status' => 500 ) );
-			return $contents;
+		if ( is_wp_error( $item ) ) {
+			return $item;
 		}
 
-		$collection_data = array_merge( $config, $contents );
-		return rest_ensure_response( $collection_data );
+		return $item;
+	}
+
+	/*
+	* Prepare a single collection output for response.
+	*
+	* @since 6.5.0
+	*
+	* @param WP_Font_Collection $collection Collection object.
+	* @param WP_REST_Request    $request    Request object.
+	* @return array|WP_Error
+	*/
+	public function prepare_item_for_response( $collection, $request ) {
+		$fields = $this->get_fields_for_response( $request );
+		$item   = array();
+
+		$config_fields = array( 'slug', 'name', 'description' );
+		foreach ( $config_fields as $field ) {
+			if ( in_array( $field, $fields, true ) ) {
+				$item[ $field ] = $collection->$field;
+			}
+		}
+
+		$data_fields = array( 'font_families', 'categories' );
+		if ( in_array( 'font_families', $fields, true ) || in_array( 'categories', $fields, true ) ) {
+			$content = $collection->get_content();
+
+			// If there was an error getting the collection data, return the error.
+			if ( is_wp_error( $content ) ) {
+				$content->add_data( array( 'status' => 500 ) );
+				return $content;
+			}
+
+			foreach ( $data_fields as $field ) {
+				if ( in_array( $field, $fields, true ) ) {
+					$item[ $field ] = $content[ $field ];
+				}
+			}
+		}
+
+		return rest_ensure_response( $item );
+	}
+
+	/**
+	 * Retrieves the font collection's schema, conforming to JSON Schema.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @return array Item schema data.
+	 */
+	public function get_item_schema() {
+		if ( $this->schema ) {
+			return $this->add_additional_fields_schema( $this->schema );
+		}
+
+		$schema = array(
+			'$schema'    => 'http://json-schema.org/draft-04/schema#',
+			'title'      => 'font-collection',
+			'type'       => 'object',
+			'properties' => array(
+				'slug'          => array(
+					'description' => __( 'Unique identifier for the font collection.', 'gutenberg' ),
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit' ),
```

Comment:
Is there a `view` context for font collections? Based on the permissions check, it doesn't look like it.
```suggestion
					'context'     => array( 'edit' ),
```

Reply:
replaced all occurrences here: https://github.com/WordPress/gutenberg/pull/58222/commits/e1062356e52edf994994461805e78f7c8fb8fd1e

Reply:
> Is there a view context for font collections?

All the core endpoints seem to be defaulting to `view` so following that pattern seems to be the best path.

It seems like we misinterpreted the meaning of the `context` parameter.  If we follow the explanation from https://make.xwp.co/2017/07/25/defining-context-in-the-wp-rest-api/  we should revert https://github.com/WordPress/gutenberg/commit/e1062356e52edf994994461805e78f7c8fb8fd1e
and use only `view` for the font collection because font collections cannot be "edited".

Reply:
Good call and thanks for that reference--makes more sense now. I'm also updating the other controllers here: https://github.com/WordPress/gutenberg/pull/58287

Reply:
added `context => array('view', 'edit', 'embed')` here